### ISSUE TITLE
Import of template fail

### ISSUE THREAD

**Issue by gmourani** [Cacti/plugin_reportit#80]
I've created a new template then tried to import it after a fresh re-install of Cacti with reportit without success.

Here the error message during import time:
Checksun error with Template 1 in xml file.

<img width="599" alt="screen shot 2018-11-27 at 4 04 32 pm" src="https://user-images.githubusercontent.com/20328799/49111600-9a0a1600-f25e-11e8-9fcb-70613fdeaff9.png">

Regards,

**Comment by netniV** [comment]
Was it the same version of reportit?  I made some radical changes to the import facility.

**Comment by gmourani** [comment]
Cacti 1.1.38
Reportit upcoming 1.1.3

I'll download the latest dev code today Nov 28 2018 and test again.

**Comment by gmourani** [comment]
I've tested with latest dev code now and still the same problem with checksum error.

**Comment by netniV** [comment]
Can you post your template or email it to me (netniv@cacti.net) and I can review it.

**Comment by gmourani** [comment]
Hello Mark,

Please, find attached to this message the template, thanks.

Regards,

**Comment by netniV** [comment]
I don't see an attachment on GitHub, does that mean you also emailed it to me?

**Comment by gmourani** [comment]
Hello,

I’ve replied to all on the email and also on GitHub. I think GitHub block this kind of attachement (XML). If you email me directly with your email, I’ll be able to reply to you with the attached XML file.

Regards,

**Comment by netniV** [comment]
It was in the reply before last (eg, 3 emails ago).

**Comment by gmourani** [comment]
Sent, thanks.

**Comment by netniV** [comment]
I've tried to replicate this issue but I have been unable to.  If you are still getting this @gmourani it would be nice to have a dump of your database to see why you are....

**Comment by netniV** [comment]
Thanks, I have a copy of that now, I'll see what I can spot as being the problem.

**Comment by gmourani** [comment]
Very nice, let me know if you need something else, thanks.

**Comment by netniV** [comment]
The DB doesn't appear to have the exported template...

**Comment by gmourani** [comment]
I can't import the template. So it couldn't have something to export !

**Comment by netniV** [comment]
Ah, I expect the original DB that held the template so I could run checks during import/export.

**Comment by gmourani** [comment]
I'm not awar of some initial template into the DB!

**Comment by netniV** [comment]
So where did the export come from?

**Comment by gmourani** [comment]
From a previous template that I've manually created before exporting as xml file and that I've tried to import from by re-installing Cacti and plugins.

I've attached the exported xml template in this message, if this could help you.
[Network-Traffic.xml.zip](https://github.com/Cacti/plugin_reportit/files/3407710/Network-Traffic.xml.zip)

**Comment by netniV** [comment]
Right, so I exported a template that I had, and then reimported it.  However, my import didn't have an issue.

**Comment by netniV** [comment]
So, I went and disabled the checksum check and imported your template, then I exported it again and tried to import.  Again, no errors.  I wondered if you had an older version of the plugin where the problem existed but I ruled that out as the version says the same (unless its a patch within that version).  But for now I can't reproduce this.

**Comment by netniV** [comment]
```xml
<?xml version="1.0" encoding="UTF-8"?>
<report_templates>
<report_template>
  <reportit>
    <version>1.1.3</version>
    <type>1</type>
    <hash>a5e27cd47092b42936c212e6789996bf</hash>
  </reportit>
  <settings>
    <id>22</id>
    <name>5 Minute Report</name>
    <user_id>0</user_id>
    <modified_by>0</modified_by>
    <last_modified>2018-10-03 13:20:30</last_modified>
    <author>admin</author>
    <version>1.0</version>
    <description>My desc</description>
    <pre_filter/>
    <data_template_id>1</data_template_id>
    <locked>on</locked>
    <enabled/>
    <export_folder/>
  </settings>
  <measurands>
		</measurands>
  <variables>
		</variables>
  <data_source_items>
    <data_source_item>
      <id>0</id>
      <template_id>22</template_id>
      <group_id>0</group_id>
      <data_template_id>0</data_template_id>
      <data_source_name>5min_cpu</data_source_name>
      <data_source_alias/>
      <data_source_title/>
      <data_source_mp>1</data_source_mp>
    </data_source_item>
  </data_source_items>
</report_template></report_templates>
```

**Comment by gmourani** [comment]
Thanks for the feedback. To eliminate the possibility that it's a patch within that version, I can download the latest available version on GIT and taste with it to see if I get the same result?

**Comment by netniV** [comment]
You could copy the above XML directly from the page.  However, I wouldn't recommend the tasting of git as that's going to take quite a few bytes to eat (seriously though you can replace the code with a git cloned one as long as the directory it the same)
